zone/embparser.cpp

```diff
@@ -2060,14 +2085,16 @@ void PerlembParser::ExportEventVariables(
 				ExportVar(package_name.c_str(), "healing_received", sep.arg[8]);
 			}
 
-			if (extra_pointers && extra_pointers->size() >= 1) {
-				Corpse* corpse = std::any_cast<Corpse*>(extra_pointers->at(0));
-				if (corpse) {
-					ExportVar(package_name.c_str(), "killed_corpse_id", corpse->GetID());
-					ExportVar(package_name.c_str(), "killed_x", corpse->GetX());
-					ExportVar(package_name.c_str(), "killed_y", corpse->GetY());
-					ExportVar(package_name.c_str(), "killed_z", corpse->GetZ());
-					ExportVar(package_name.c_str(), "killed_h", corpse->GetHeading());
+			if (!npc_mob->IsMerc()) {
```

Comment:
This check shouldn't be necessary now since corpse will be nullptr for mercs and not do anything